### PR TITLE
[MODULAR] Replacement syringe gun crate

### PR DESCRIPTION
**Pull request by SpookyTheFox**
## About The Pull Request
A replacement syringe gun.

## How This Contributes To The Skyrat Roleplay Experience
Paramedic took the syringe gun to cryo? Clown stole the other one? Now you can order a replacement!
Due to skyrat's long rounds and high population, map spawn items tend to go missing and stay gone before those that are supposed to have them, get them. Some maps only have one of these spawn. Even when people don't intend to hog it for the round, it tends to disappear into the clutter of the medical storage room. The cost has been set to the same as the cell powered medigun, normally a 20 minute cool down order. This ensures an opportunity cost of not ordering anything else like modsuits or pen crates for a long period of time, or needing the station to cover the cost.

## Changelog

:cl:
add: single replacement syringe gun crate
/:cl:

Verified working on local test.

### CODE REVIEW

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 1]
```diff
@@ -107,6 +107,14 @@
 					/obj/item/weaponcell/medical/toxin)
 	crate_name = "medicell replacement crate"
 
+/datum/supply_pack/medical/syringe_gun
+	name = "Syringe Gun"
+	desc = "A replacement medical syringe gun"
```
Comment (Zonespace27) [inline modular_skyrat/modules/cargo/code/packs.dm]:
```suggestion
	desc = "A replacement medical syringe gun."
```

File: modular_skyrat/modules/cargo/code/packs.dm [hunk 2]
```diff
@@ -107,6 +107,14 @@
...
+	cost = CARGO_CRATE_VALUE * 30
+	access = ACCESS_MEDICAL
+	contains = list(/obj/item/gun/syringe)
+	crate_name = "replacement syringe gun crate."
```
Comment (Zonespace27) [inline modular_skyrat/modules/cargo/code/packs.dm]:
```suggestion
	crate_name = "replacement syringe gun crate"
```

**Comment by softcerv** [comment]
I'd advise maybe moving this to goodies or some other category besides medical, mostly because medical now has SmartDart guns which function as syringe guns but without the ability to use harmful chemicals and piercing syringes (they can also be printed at the medical prolathe,) making the standard syringe guns more of an antag/self defense tool.

File: modular_skyrat/modules/cargo/code/packs.dm (hunk 1 above)
Comment (SpookyTheFox) [reply modular_skyrat/modules/cargo/code/packs.dm]: I was trying to keep consistency with the other ones, but they really aren't super consistent.

**Comment by SpookyTheFox** [comment]
> I'd advise maybe moving this to goodies or some other category besides medical, mostly because medical now has SmartDart guns which function as syringe guns but without the ability to use harmful chemicals and piercing syringes (they can also be printed at the medical prolathe,) making the standard syringe guns more of an antag/self defense tool.

I considered that. It's really only valuable to someone with not only access to chemistry equipment, but then also the funds to buy it. The goodie pack route, the cost would be on the purchaser, and the only reasonable values do not jive with what I expect them to earn from bounties in a reasonable amount of time. They can buy shotguns and print reagent dart shells from an autolathe if they really want to go that route, but it just does not make sense in terms of time to earn the money, ways it can go wrong, and final performance. Bounties in general really need attention, but that's way beyond the scope of this PR.

I think putting it in goodie packs also legitimizes it as a weapon, and from an IC standpoint, it's a medical tool not for personal use.

I probably would have passed on making this PR entirely since the rapid syringe gun exists in code and there's supposed to be a route to obtain it even as a non-antag, But you actually can't get it, or anything from illegal tech. Our tech nodes are a complete mess in need of balance, QoL, and bug fixes, with no pending plans for any of those.

**Comment by softcerv** [comment]
>I think putting it in goodie packs also legitimizes it as a weapon, and from an IC standpoint, it's a medical tool not for personal use.

I'm going to be honest, 9/10 times it usually is used as one. Especially now that SmartDarts are an option here. The main reason why I'm against it being a departmental order is that it feels like a very personal oriented purchase, especially when compared to other things like Mediguns and MODsuits, I don't feel like this is something I'd see a CMO buy for their department unless they wanted to arm up.

**Comment by MassacreDemigirl** [comment]
medical has really limited stuff to buy that is actually worthwhile, I say its a good addition

**Comment by OzzyMcPseudonym** [comment]
In all honesty I am all for the goodie pack route. Something akin to the revolver cost would be pretty balanced if you ask me (mostly cause you can just fill an armor-piercing shot with 2 Anacea and 8 Lexorin and practically crit or kill someone off of that alone, that stuff should probably be expensive.) The big thing to consider is "How does the syringe gun compare to other tools that medical has" (Bath salts, meth, ephedrine, smoke and foam, Laser Scalpel, etc). As far as accessibility is concerned it's a rather powerful tool that any doctor can reasonably use if they know a recipe to Chloral Hydrate or Super Melatonin or something else that debilitates or kills targets. 

It applies the darts to one target instead of an AOE, which in my opinion makes it more valuable than smoke and foam because it has less chance of accidentally hitting someone that you didn't mean to hit. Not only more valuable from a "Don't hurt innocents" perspective but also from a policy perspective. It makes much more sense to make a sedative dart IC than to make a whole-ass sedative foam grenade. Without a syringe gun medics are encouraged to instead use Melatonin foam with a Modafinil pill on themselves to prevent themself from falling asleep to their own grenade. Syringe guns make the chem meta more IC reasonable and also provide doctors with a means of targeted self-defense if some DEsword guy is slashing up people in the bay for some reason.

**Comment by YakumoChen** [comment]
> I probably would have passed on making this PR entirely since the rapid syringe gun exists in code and there's supposed to be a route to obtain it even as a non-antag, But you actually can't get it, or anything from illegal tech. Our tech nodes are a complete mess in need of balance, QoL, and bug fixes, with no pending plans for any of those.

https://github.com/Skyrat-SS13/Skyrat-tg/pull/2285

The RSG move to illegal is pretty justified. As was said, syringes can be a one shot kill. Having a mass produced, rapid fire, one shot one kill guns is not something we want to have readily available most rounds.

**Comment by MassacreDemigirl** [comment]
great point @YakumoChen however we have 10mm makarovs on sale as goodies.

**Comment by YakumoChen** [comment]
that's fine, but not a point to justify Rapid Syringe guns being readily obtainable. Out of the scope of the PR, regardless.

**Comment by MassacreDemigirl** [comment]
ah I was not aware we were talking about making RSG buyable, but having a way to replace normal syringer is nice

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by Gandalf2k15** [comment]
I see no harm given the way we are planning to go with cargo.

**Pull request merged**